Source/Kesmai.Server/Game/Items/Equipment/Weapons/Weapon.cs

```diff
@@ -48,12 +48,37 @@ public virtual double GetAttackBonus(MobileEntity attacker, MobileEntity defende
 		/// </remarks>
 		public override int CalculateBlockingBonus(ItemEntity item)
 		{
-			if (item is IWeapon weapon && weapon.Flags.HasFlag(WeaponFlags.Projectile))
-				return ProjectileProtection;
+			var flags = WeaponFlags.Bashing;
```

Comment:
not sure if this is an issue, but if item isn't a weapon you still return bashing + base armor, should this return 0 instead?

Reply:
initializes as bashing so things that don't use weapons like lairs have a damage type, if the creature weilds a weapon, overwrite that with the weapon flags

Reply:
makes sense to some extent, but we are now getting bashed by claws instead of slashed etc.  probably fine in the scheme of things and prevents the need to set damage types on custom attacks